src/app/features/genes/components/gene-nominated-targets/gene-nominated-targets.component.ts

```diff
@@ -120,37 +120,10 @@ export class GeneNominatedTargetsComponent implements OnInit {
         inputDataArray = this.commaFlattenArray(inputDataArray);
 
         // Populate targetNomination display fields
-        de.teams_display_value = '';
-        if (teamsArray.length) {
-          de.teams_display_value = teamsArray
-            .filter(this.getUnique)
-            .sort((a: string, b: string) => a.localeCompare(b))
-            .join(', ');
-        }
```

Comment:
Moved repeated logic into `formatDisplayValue` method

Reply:
I see you renamed it to getCommaSeparatedStringOfUniqueSortedValues.  Looks good!